awx/main/analytics/collectors.py

```diff
@@ -86,7 +86,6 @@ def counts(since, **kwargs):
     inv_counts['smart'] = inv_counts.get('smart', 0)
     counts['inventories'] = inv_counts
     
-    counts['unified_job'] = models.UnifiedJob.objects.exclude(launch_type='sync').count() # excludes implicit project_updates
     counts['active_host_count'] = models.Host.objects.active_count()   
```

Comment:
@wenottingham hm, how would we use the the active host count? From code it does:

```
self.order_by().exclude(inventory_sources__source='tower').values('name').distinct().count()
```

So unique host count across all inventories? Not sure why we're excluding tower source. We can get this, only if you're running some jobs on those inventories.

Reply:
And how this relates to `inventory_counts` below

Reply:
It's a de-duped count of hosts across inventories. The reason that tower source is included is because hosts sourced from another Tower instance don't "count" for licensing reasons (as they are already counted under the master.)

Reply:
Yea, I just wont' drop this for now